Assets/Scripts/PlatformOneTouch.cs

```diff
@@ -0,0 +1,20 @@
+using System.Collections;
+using System.Collections.Generic;
+using System.Threading;
+using Unity.VisualScripting;
+using UnityEngine;
+
+public class PlatformOneTouch : MonoBehaviour
+{
+    short rnd;
```

Comment:
V takomto prípade kľudne použi `int`, rozdiel v performance nebude žiadny (myslím, že moderné procesory sú aj lepšie optimalizované na int a v niektorých prípadoch bude ten short aj tak paddingnutý na int), plus je to taká konvencia, že pokiaľ to fakt nie je potrebné (databázy napr.) tak nepoužívame iné dátové typy.

Reply:
Ok, budem teda používať inty